package.json

```diff
@@ -7,7 +7,8 @@
   ],
   "scripts": {
     "postinstall": "patch-package",
-    "build": "turbo run build --continue --filter=\"!@dao-dao/storybook\"",
+    "build:original": "turbo run build --continue --filter=\"!@dao-dao/storybook\"",
```

Comment:
I added this on debugging stage, i dont know if we want to preserve this or just get rid of build, renaming build:original back to build.

Reply:
I have no opinion on this.  It works as it is right now.  I suspect that decision would be a developer decision and not mine to make.

Reply:
@akure Do you have an opinion on this?